### PR TITLE
Kd format indiv record vc

### PR DESCRIPTION
**Pull request by karin-**
@wtachau the veryLightBlue color for section headers looked weird, so I changed it to lightBlue, but that also looks weird. color suggestions?

### CODE REVIEW

File: IndividualRecordViewController.m [hunk 1]
```diff
@@ -33,11 +35,25 @@ - (UITableView*)recordTableView {
...
   _recordTableView.delegate = self;
   _recordTableView.dataSource = self;
   
   UITextView *defaultText = [[UITextView alloc] init];
-  defaultText.text = NSLocalizedString(@"RECORD_DISCLAIMER_TEXT", nil);
+  NSString* disclaimerText = NSLocalizedString(@"RECORD_DISCLAIMER_TEXT", nil);
+  NSMutableAttributedString* attrString = [[NSMutableAttributedString alloc] initWithString:disclaimerText];
+  NSRange boldRange1 = [disclaimerText rangeOfString:@"Note"];
+  NSRange boldRange2 = [disclaimerText rangeOfString:@"Key"];
+  NSRange italicRange = NSMakeRange(boldRange1.location + boldRange1.length + 1, 206);
```
Comment (aarondaub) [inline IndividualRecordViewController.m]: 206 seems like a magic number to me, what it is it in reference to?

File: Medlink.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -277,7 +277,7 @@
 		B87306541B5D853B0019B58F /* LoginTextField.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = LoginTextField.h; sourceTree = "<group>"; };
 		B87306551B5D853B0019B58F /* LoginTextField.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = LoginTextField.m; sourceTree = "<group>"; };
 		B87306581B5D98AE0019B58F /* IndividualRecordViewController.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; name = IndividualRecordViewController.h; path = ../IndividualRecordViewController.h; sourceTree = "<group>"; };
-		B87306591B5D98AE0019B58F /* IndividualRecordViewController.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; name = IndividualRecordViewController.m; path = ../IndividualRecordViewController.m; sourceTree = "<group>"; };
+		B87306591B5D98AE0019B58F /* IndividualRecordViewController.m */ = {isa = PBXFileReference; fileEncoding = 4; indentWidth = 2; lastKnownFileType = sourcecode.c.objc; name = IndividualRecordViewController.m; path = ../IndividualRecordViewController.m; sourceTree = "<group>"; tabWidth = 2; };
```
Comment (aarondaub) [inline Medlink.xcodeproj/project.pbxproj]: I believe our indent width is supposed to be 4, no? /cc @wtachau

File: IndividualRecordViewController.m (hunk 1 above)
Comment (karin-) [inline IndividualRecordViewController.m]: The length of a whole paragraph that should be italicized. Suggestions for alternative ways to handle this?

File: Medlink.xcodeproj/project.pbxproj (hunk 2 above)
Comment (wtachau) [inline Medlink.xcodeproj/project.pbxproj]: incorrect @aarondaub

File: IndividualRecordViewController.m (hunk 1 above)
Comment (wtachau) [inline IndividualRecordViewController.m]: you could find the range by looking up the index of the next text you know not to be italicized yada yada yada but this is a prototype. I say it's fine.

**Comment by wtachau** [comment]
:+1:

**Pull request merged**